.vtexignore

```diff
@@ -0,0 +1,11 @@
+.git/
+react/node_modules/
+.eslintrc
+.gitignore
+package.json
+README.md
+CHANGELOG.md
+react/**/__tests__/**
+react/**/__mocks__/**
+react/**/*.test.ts
```

Comment:
`ts(x)`?

Reply:
Actually the `react/**/__tests__/**` its already doing the job, so I decided to remove this line.